resqpy/rq_import/_add_surfaces.py

```diff
@@ -106,10 +111,24 @@ def _add_single_surface(model, surf_file, surface_file_format, surface_role, qua
     surface.write_hdf5()
 
     if make_horizon_interpretations_and_features:
-        feature = rqo.GeneticBoundaryFeature(model, kind = 'horizon', feature_name = short_name)
-        feature.create_xml()
-        interp = rqo.HorizonInterpretation(model, genetic_boundary_feature = feature, domain = 'depth')
-        interp_root = interp.create_xml()
+        if interpretation_type == 'horizon':
+            feature = rqo.GeneticBoundaryFeature(model, kind = 'horizon', feature_name = short_name)
+            feature.create_xml()
+            interp = rqo.HorizonInterpretation(model, genetic_boundary_feature = feature, domain = 'depth')
+            interp_root = interp.create_xml()
+        elif interpretation_type == 'fault':
+            feature = rqo.TectonicBoundaryFeature(model, kind = 'fault', feature_name = short_name)
+            feature.create_xml()
+            interp = rqo.FaultInterpretation(model,
+                                             tectonic_boundary_feature = feature,
+                                             domain = 'depth',
+                                             is_normal = True)
```

Comment:
Let's update the is_normal setting to have control from another argument please.